gcloud/logging/client.py

```diff
@@ -56,6 +57,7 @@ def logger(self, name):
         """
         return Logger(name, client=self)
 
+<<<<<<< HEAD
```

Comment:
Yup.  Must've undone the delete accidentally while saving, as I had just run `tox -e cover`.